app/BUILD.bazel

```diff
@@ -510,7 +510,7 @@ kt_android_library(
     custom_package = "org.oppia.android.app",
     deps = [
         ":dagger",
-        "//model",
```

Comment:
You will have to keep the //model dependency until we have migrated all the other dependencies as well.

Reply:
Done PTAL

Reply:
@fsharpasharp 
PTAL